src/twisted/internet/task.py

```diff
@@ -860,14 +861,14 @@ def react(main, argv=(), _reactor=None):
 
     The following demonstrates the signature of a C{main} function which can be
     used with L{react}::
-          def main(reactor, username, password):
-              return defer.succeed('ok')
+          async def main(reactor, username, password):
+              return "ok"
 
-          task.react(main, ('alice', 'secret'))
+          task.react(main, ("alice", "secret"))
 
-    @param main: A callable which returns a L{Deferred}. It should
-        take the reactor as its first parameter, followed by the elements of
-        C{argv}.
+    @param main: A callable which returns a L{Deferred} or
+        C{async def} C{coroutine}. It should take the reactor as its first
```

Comment:
Instead of "C{async def} C{coroutine}" I'd just say "coroutine".

Reply:
```suggestion
        coroutine. It should take the reactor as its first
```